distribution/docs/src/main/resources/_contents/introductioncontents.adoc

```diff
@@ -0,0 +1,105 @@
+{branding} is a free and open-source common data layer that abstracts services and business logic from the underlying data structures to enable rapid integration of new data sources.
+
+Licensed under http://www.gnu.org/licenses/gpl.html[LGPL], {branding} is an interoperability platform that provides secure and scalable discovery and retrieval from a wide array of disparate sources.
+{branding} is built to "unzip and run" while having the ability to scale to large enterprise systems.
+
+Distributed Data Framework ({branding}) is an agile and modular integration framework.
+It is primarily focused on data integration, enabling clients to insert, query and transform information from disparate data sources via the {branding} Catalog.
+A Catalog API allows integrators to insert new capabilities at various stages throughout each operation.
+
+{branding} is designed with the following architectural qualities to benefit integrators.
+
+* Standardization
```

Comment:
It's also a bit strange to have full sentences in some cases, and a single word in others.